flow360/component/flow360_params/solvers.py

```diff
@@ -392,7 +394,7 @@ class HeatEquationSolver(Flow360BaseModel):
 
     absolute_tolerance: Optional[PositiveFloat] = pd.Field(alias="absoluteTolerance")
     equation_eval_frequency: Optional[PositiveInt] = pd.Field(alias="equationEvalFrequency")
-    linear_solver: Optional[LinearSolver] = pd.Field(alias="linearSolver", default=LinearSolver())
+    linear_solver: Optional[LinearSolver] = pd.Field(alias="linearSolver")
```

Comment:
why removed default?

Reply:
Added it back, deleting duplicate defaults is now handled as a schema postprocessing step